src/test/fixtures/blueprints.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  GetBlueprintsApiResponse,
+  CreateBlueprintResponse,
+} from '../../store/imageBuilderApi';
+
+export const mockBlueprintsCreation: CreateBlueprintResponse[] = [
+  {
+    id: '677b010b-e95e-4694-9813-d11d847f1bfc',
```

Comment:
Could we use (visibly) unique UUIDs for all of these? In this case I think we *haven't* done the best job in our code base so shame on us, but going forward we should probably use something like https://www.uuidgenerator.net/version4 when we need a new UUID.

Reply:
I think it a different scope, most of the fixtures use hard coded uuid, so this is a refactor for the codebase.